File: Medyana/Medyana.Application/ClientApp/src/app/models/equipment.ts
export interface Result {
    id: number;
    name: string;
    provideDate?: any;
    quantity: number;
    unitPrice: number;
    usageRate: number;
    clinicId: number;
    clinic?: any;
    createdDate: Date;
    updateDate?: Date;
    deletedDate?: Date;
    isActive: boolean;
}

export interface Equipment {
    result: Result[];
    isSucceed: boolean;
    errorMessage?: any;
}

File: Medyana/Medyana.Domain/Entities/Equipment.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Medyana.Domain.Entities
{
    public sealed class Equipment : BaseEntity
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }

        public DateTime? ProvideDate { get; set; }

        [Required]
        [Range(1, int.MaxValue)]
        public int Quantity { get; set; }

        [Required]
        [Range(0.01, double.MaxValue, ErrorMessage = "The field {0} must be greater than {1}.")]
        public decimal UnitPrice { get; set; }

        [Required]
        [Range(0.00, 100, ErrorMessage = "The field {0} must be greater than {1}.")]
        public decimal UsageRate { get; set; }

        public int ClinicId { get; set; }
        public Clinic Clinic { get; set; }
    }
}